pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/basicsecretmanagersecret/create.yaml

```diff
@@ -24,7 +24,4 @@ spec:
   replication:
     userManaged:
       replicas:
-      - location: us-central1
-  topics:
```

Comment:
Can we keep topics?  Any reason to drop test coverage here?

Reply:
Ah - maybe because we moved it to the full test?

Reply:
Right. Topic is an optional field. I recommend only put required fields in the basic test suite, in which way it is easier to tell how to configure a field.